R/topics.R

```diff
@@ -196,7 +196,7 @@ content_info <- function(content_entry, index, pkg, section) {
     tibble::tibble(
       path = rd_href,
       aliases = rd_aliases,
-      name = list(content_entry = NULL),
+      name = content_entry,
```

Comment:
shouldn't it be null?

Reply:
Well, it has to be something because it's collapsed into a data frame at the next stage, and it only appears to be used for de-duplication, so I think this is a reasonable default.